kofta/src/app/components/Button.tsx

```diff
@@ -87,7 +87,7 @@ export const Button: React.FC<
             variant === "small"
               ? "py-1 px-2 w-max"
               : variant === "slim"
-              ? "max-w-md ml-auto mr-auto py-2.5 px-12"
+              ? "max-w-2xl ml-auto mr-auto py-2.5 px-12"
```

Comment:
I like that the buttons are not full width on desktop atm

Reply:
Sounds good, I will remove this change from the pull request